src/assets/global.css

```diff
@@ -45,6 +45,7 @@ body.loading.message:after {
     var(--w) calc(-1 * var(--w)) 0 var(--c), calc(-1 * var(--w)) var(--w) 0 var(--c),
     0 var(--w) 0 var(--c), 0 calc(-1 * var(--w)) 0 var(--c), var(--w) 0 0 var(--c),
     calc(-1 * var(--w)) 0 0 var(--c);
+  width: 100%;
```

Comment:
so it's centred on big screens or zoomed-out views

Reply:
is there a tool that helped you write that text-shadow?

Reply:
not really, i remember seeing something in a CodePen linked from a newsletter from css-tricks or smashingmagazine i think.  it's simple really, it's just [8 shadows, with 0 blur, each offset in a different direction](https://codepen.io/hchiam/pen/vYGWPQr) to give the illusion of an outline.